src/entity/user/UserStats.ts

```diff
@@ -52,6 +52,7 @@ import { ghostUser } from '../../common';
 })
 export class UserStats {
   @ViewColumn()
+  @Index('IDX_user_stats_id')
```

Comment:
Is the plan locally or production?

Reply:
Production. Locally I never had the issue because there were only 10 rows to select from, so it only took a few ms 😬